xDrip/Scenes/Entries/EntriesList/EntriesListViewController.swift

```diff
@@ -59,9 +66,58 @@ class EntriesListViewController: UIViewController, EntriesListDisplayLogic {
         interactor?.doLoad(request: request)
     }
     
+    @IBAction private func onCancelButtonTap() {
+        let request = EntriesList.Cancel.Request()
+        interactor?.doCancel(request: request)
+    }
+    
+    @objc
+    private func onEditButtonTap() {
+        isEdit.toggle()
+        setupRightBarButtonItem()
+        tableView.isEditing = isEdit
+    }
+    
     // MARK: Display
     
     func displayLoad(viewModel: EntriesList.Load.ViewModel) {
+        sectionViewModels = viewModel.items
+        
+        tableController.reload(with: viewModel.items)
```

Comment:
Move `tableView.reloadData()` to `tableController`, because you're splitting up common logic. To achieve this, table controller should store a weak reference to the table.